core/test/functional/routes/api/users_spec.js

```diff
@@ -39,10 +39,10 @@ describe('User API', function () {
             .then(function (_user2) {
                 author = _user2;
 
-                // create inactive user
+                // create contributor
                 return testUtils.createUser({
-                    user: testUtils.DataGenerator.forKnex.createUser({email: 'test+3@ghost.org', status: 'inactive'}),
-                    role: testUtils.DataGenerator.Content.roles[2]
+                    user: testUtils.DataGenerator.forKnex.createUser({email: 'test+4@ghost.org'}),
+                    role: testUtils.DataGenerator.Content.roles[4]
```

Comment:
why do we need this change? 🤔

Reply:
ah whoops - must have not reverted _all_ of the changes there 🤦‍♂️